jcb_client_init.py

```diff
@@ -185,7 +185,8 @@ def clone_or_update_repos(jcb_apps: typing.Dict[str, typing.Dict[str, typing.Any
 
             # Clone the repository
             command_string = ' '.join(git_clone)
-            write_message(f'Cloning {app} with command: {command_string}')
+            #write_message(f'Cloning {app} with command: {command_string}')
+            print(f'Cloning {app} with command: {command_string}')
```

Comment:
Any particular reason why this had to be changed? `write_message` is used in other places

Reply:
don't spend too much time on it but if it's a python version issue or something that we can easily explain that would be helpful